validphys2/src/validphys/plotoptions/kintransforms.py

```diff
@@ -267,6 +267,15 @@ def new_labels(self, *old_labels):
         return (r'$\eta_{dijet}$', '$p_{T,avg}$ (GeV)', r'$\sqrt{s}$ (GeV)')
 
 
+class dis_q2_scale(DISXQ2MapMixin):
+    def __call__(self, k1, k2, k3):
+        ecm = np.sqrt(k2 / (k1 * k3))
+        return k1, k2, np.ceil(ecm)
+
+    def new_labels(self, *old_labels):
+        return ('$x$', '$Q^2$ (GeV)', r'$\sqrt{s}$ (GeV)')
+
+
```

Comment:
```suggestion
```

We want to remove this module!
You should not need a kinematic override. Instead of doing this the `kinematics.yaml` file should be changed.

Reply:
done in 5e5208feb